corehq/apps/locations/models.py

```diff
@@ -116,6 +116,18 @@ class LocationQueriesMixin(object):
     def location_ids(self):
         return self.values_list('location_id', flat=True)
 
+    def couch_locations(self, wrapped=True):
+        """
+        Returns the couch locations corresponding to this queryset.
+        If you're calling this, take a long hard look at yourself and ask why.
+        Do you really need couch locations here?
+        """
```

Comment:
dang that looks annoying. possibly I haven't seen these since we upgraded django?